### PR TITLE
Fix Javascript injection & ReDoS vulnerabilities 

### PR DESCRIPTION
**Pull request by scottinet**
:warning: depends on https://github.com/kuzzleio/kuzzle-plugin-cluster/pull/99

# Description

Fix harmful uses of JS keywords in requests made to Kuzzle, due to a lack of distrust about external data.

Example: an index named `constructor` is stored as a key, and then used like this: `this.someCacheObject[index]`, effectively invoking Object.constructor, which is a Javascript reserved keyword.

This issue is solved by:
- storing our RAM cache in maps
- adding a new utility module made to safely check or access to object properties (with 50x to 300x better performances than the equivalent lodash methods)

### CODE REVIEW

File: lib/api/kuzzle.js [hunk 1]
```diff
@@ -342,7 +341,9 @@ function dumpAndExit(kuzzle, suffix) {
     runShutdown(kuzzle);
   }
 
-  kuzzle.janitor.dump(suffix).finally(() => runShutdown(kuzzle));
+  kuzzle.janitor.dump(suffix)
+    .catch(() => {}) // do nothing, prevents unhandled rejections
```
Comment (kuzzle) [inline lib/api/kuzzle.js]: CRITICAL Add a nested comment explaining why this function is empty or complete the implementation. [rule](https://sonarqube.kaliop.net/coding_rules#rule_key=javascript%3AS1186)
MAJOR Either remove or fill this block of code. [rule](https://sonarqube.kaliop.net/coding_rules#rule_key=javascript%3AEmptyBlock)

File: lib/api/core/hotelClerk.js [hunk 2]
```diff
@@ -133,13 +135,13 @@ class HotelClerk {
...
       roomId = request.input.body.roomId,
-      room = this.rooms[roomId];
+      room = this.rooms.get(roomId);
 
     if (room === undefined) {
       realtimeError.throw('room_not_found', roomId);
     }
 
-    return {count: this.rooms[roomId].customers.size};
+    return {count: room.customers.size};
```
Comment (kuzzle) [inline lib/api/core/hotelClerk.js]: MAJOR TypeError can be thrown as "room" might be null or undefined here. [rule](https://sonarqube.kaliop.net/coding_rules#rule_key=javascript%3AS2259)

**Comment by kuzzle** [comment]
SonarQube analysis reported 13 issues
* ![CRITICAL][CRITICAL] 1 critical
* ![MAJOR][MAJOR] 2 major
* ![MINOR][MINOR] 1 minor
* ![INFO][INFO] 9 info

Watch the comments in this conversation to review them.

#### 10 extra issues

Note: The following issues were found on lines that were not modified in the pull request. Because these issues can't be reported as line comments, they are summarized here:

1. ![MINOR][MINOR] [elasticsearch.js#L399](https://github.com/kuzzleio/kuzzle/blob/b2bb90d4e4c58472c03549315506f1f973716884/lib/services/elasticsearch.js#L399): Group all shorthand properties at either the beginning or end of this object declaration. [rule](https://sonarqube.kaliop.net/coding_rules#rule_key=javascript%3AS3499)
1. ![INFO][INFO] [adminController.js#L57](https://github.com/kuzzleio/kuzzle/blob/b2bb90d4e4c58472c03549315506f1f973716884/lib/api/controllers/adminController.js#L57): Complete the task associated to this TODO comment. [rule](https://sonarqube.kaliop.net/coding_rules#rule_key=javascript%3AS1135)
1. ![INFO][INFO] [baseController.js#L648](https://github.com/kuzzleio/kuzzle/blob/b2bb90d4e4c58472c03549315506f1f973716884/lib/api/controllers/baseController.js#L648): Complete the task associated to this TODO comment. [rule](https://sonarqube.kaliop.net/coding_rules#rule_key=javascript%3AS1135)
1. ![INFO][INFO] [documentController.js#L430](https://github.com/kuzzleio/kuzzle/blob/b2bb90d4e4c58472c03549315506f1f973716884/lib/api/controllers/documentController.js#L430): Complete the task associated to this TODO comment. [rule](https://sonarqube.kaliop.net/coding_rules#rule_key=javascript%3AS1135)
1. ![INFO][INFO] [documentController.js#L480](https://github.com/kuzzleio/kuzzle/blob/b2bb90d4e4c58472c03549315506f1f973716884/lib/api/controllers/documentController.js#L480): Complete the task associated to this TODO comment. [rule](https://sonarqube.kaliop.net/coding_rules#rule_key=javascript%3AS1135)
1. ![INFO][INFO] [notifier.js#L288](https://github.com/kuzzleio/kuzzle/blob/b2bb90d4e4c58472c03549315506f1f973716884/lib/api/core/notifier.js#L288): Complete the task associated to this TODO comment. [rule](https://sonarqube.kaliop.net/coding_rules#rule_key=javascript%3AS1135)
1. ![INFO][INFO] [kuzzle.js#L55](https://github.com/kuzzleio/kuzzle/blob/b2bb90d4e4c58472c03549315506f1f973716884/lib/api/kuzzle.js#L55): Complete the task associated to this TODO comment. [rule](https://sonarqube.kaliop.net/coding_rules#rule_key=javascript%3AS1135)
1. ![INFO][INFO] [elasticsearch.js#L1561](https://github.com/kuzzleio/kuzzle/blob/b2bb90d4e4c58472c03549315506f1f973716884/lib/services/elasticsearch.js#L1561): Complete the task associated to this TODO comment. [rule](https://sonarqube.kaliop.net/coding_rules#rule_key=javascript%3AS1135)
1. ![INFO][INFO] [elasticsearch.js#L1602](https://github.com/kuzzleio/kuzzle/blob/b2bb90d4e4c58472c03549315506f1f973716884/lib/services/elasticsearch.js#L1602): Complete the task associated to this TODO comment. [rule](https://sonarqube.kaliop.net/coding_rules#rule_key=javascript%3AS1135)
1. ![INFO][INFO] [elasticsearch.js#L1680](https://github.com/kuzzleio/kuzzle/blob/b2bb90d4e4c58472c03549315506f1f973716884/lib/services/elasticsearch.js#L1680): Complete the task associated to this TODO comment. [rule](https://sonarqube.kaliop.net/coding_rules#rule_key=javascript%3AS1135)

[CRITICAL]: https://sonarsource.github.io/sonar-github/severity-critical.png 'Severity: CRITICAL'
[INFO]: https://sonarsource.github.io/sonar-github/severity-info.png 'Severity: INFO'
[MAJOR]: https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR'
[MINOR]: https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR'

File: lib/api/core/storage/bootstrap/safeBootstrap.js [hunk 3]
```diff
@@ -153,32 +155,50 @@ class SafeBootstrap {
...
+
     let lock;
 
     try {
       lock = await this._indexStorage.get('config', this._LOCK_ID);
-    } catch (error) {
-      if (error.status !== 404) {
+    }
+    catch (error) {
+      if (error !== 404) {
```
Comment (Aschen) [inline lib/api/core/storage/bootstrap/safeBootstrap.js]:
```suggestion
      if (error.id !== 'services.storage.not_found') {
```

**Comment by kuzzle** [comment]
SonarQube analysis reported 10 issues
* ![MAJOR][MAJOR] 1 major
* ![INFO][INFO] 9 info

Watch the comments in this conversation to review them.

#### 9 extra issues

Note: The following issues were found on lines that were not modified in the pull request. Because these issues can't be reported as line comments, they are summarized here:

1. ![INFO][INFO] [adminController.js#L57](https://github.com/kuzzleio/kuzzle/blob/8744e26abc3d9da21ebf7ec0d620708ae31230b9/lib/api/controllers/adminController.js#L57): Complete the task associated to this TODO comment. [rule](https://sonarqube.kaliop.net/coding_rules#rule_key=javascript%3AS1135)
1. ![INFO][INFO] [baseController.js#L648](https://github.com/kuzzleio/kuzzle/blob/8744e26abc3d9da21ebf7ec0d620708ae31230b9/lib/api/controllers/baseController.js#L648): Complete the task associated to this TODO comment. [rule](https://sonarqube.kaliop.net/coding_rules#rule_key=javascript%3AS1135)
1. ![INFO][INFO] [documentController.js#L430](https://github.com/kuzzleio/kuzzle/blob/8744e26abc3d9da21ebf7ec0d620708ae31230b9/lib/api/controllers/documentController.js#L430): Complete the task associated to this TODO comment. [rule](https://sonarqube.kaliop.net/coding_rules#rule_key=javascript%3AS1135)
1. ![INFO][INFO] [documentController.js#L480](https://github.com/kuzzleio/kuzzle/blob/8744e26abc3d9da21ebf7ec0d620708ae31230b9/lib/api/controllers/documentController.js#L480): Complete the task associated to this TODO comment. [rule](https://sonarqube.kaliop.net/coding_rules#rule_key=javascript%3AS1135)
1. ![INFO][INFO] [notifier.js#L288](https://github.com/kuzzleio/kuzzle/blob/8744e26abc3d9da21ebf7ec0d620708ae31230b9/lib/api/core/notifier.js#L288): Complete the task associated to this TODO comment. [rule](https://sonarqube.kaliop.net/coding_rules#rule_key=javascript%3AS1135)
1. ![INFO][INFO] [kuzzle.js#L55](https://github.com/kuzzleio/kuzzle/blob/8744e26abc3d9da21ebf7ec0d620708ae31230b9/lib/api/kuzzle.js#L55): Complete the task associated to this TODO comment. [rule](https://sonarqube.kaliop.net/coding_rules#rule_key=javascript%3AS1135)
1. ![INFO][INFO] [elasticsearch.js#L1561](https://github.com/kuzzleio/kuzzle/blob/8744e26abc3d9da21ebf7ec0d620708ae31230b9/lib/services/elasticsearch.js#L1561): Complete the task associated to this TODO comment. [rule](https://sonarqube.kaliop.net/coding_rules#rule_key=javascript%3AS1135)
1. ![INFO][INFO] [elasticsearch.js#L1602](https://github.com/kuzzleio/kuzzle/blob/8744e26abc3d9da21ebf7ec0d620708ae31230b9/lib/services/elasticsearch.js#L1602): Complete the task associated to this TODO comment. [rule](https://sonarqube.kaliop.net/coding_rules#rule_key=javascript%3AS1135)
1. ![INFO][INFO] [elasticsearch.js#L1680](https://github.com/kuzzleio/kuzzle/blob/8744e26abc3d9da21ebf7ec0d620708ae31230b9/lib/services/elasticsearch.js#L1680): Complete the task associated to this TODO comment. [rule](https://sonarqube.kaliop.net/coding_rules#rule_key=javascript%3AS1135)

[INFO]: https://sonarsource.github.io/sonar-github/severity-info.png 'Severity: INFO'
[MAJOR]: https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR'

**Comment by kuzzle** [comment]
SonarQube analysis reported 10 issues
* ![MAJOR][MAJOR] 1 major
* ![INFO][INFO] 9 info

Watch the comments in this conversation to review them.

#### 9 extra issues

Note: The following issues were found on lines that were not modified in the pull request. Because these issues can't be reported as line comments, they are summarized here:

1. ![INFO][INFO] [adminController.js#L57](https://github.com/kuzzleio/kuzzle/blob/115be4fcfc70156d05af7560c003951de4a00690/lib/api/controllers/adminController.js#L57): Complete the task associated to this TODO comment. [rule](https://sonarqube.kaliop.net/coding_rules#rule_key=javascript%3AS1135)
1. ![INFO][INFO] [baseController.js#L648](https://github.com/kuzzleio/kuzzle/blob/115be4fcfc70156d05af7560c003951de4a00690/lib/api/controllers/baseController.js#L648): Complete the task associated to this TODO comment. [rule](https://sonarqube.kaliop.net/coding_rules#rule_key=javascript%3AS1135)
1. ![INFO][INFO] [documentController.js#L430](https://github.com/kuzzleio/kuzzle/blob/115be4fcfc70156d05af7560c003951de4a00690/lib/api/controllers/documentController.js#L430): Complete the task associated to this TODO comment. [rule](https://sonarqube.kaliop.net/coding_rules#rule_key=javascript%3AS1135)
1. ![INFO][INFO] [documentController.js#L480](https://github.com/kuzzleio/kuzzle/blob/115be4fcfc70156d05af7560c003951de4a00690/lib/api/controllers/documentController.js#L480): Complete the task associated to this TODO comment. [rule](https://sonarqube.kaliop.net/coding_rules#rule_key=javascript%3AS1135)
1. ![INFO][INFO] [notifier.js#L288](https://github.com/kuzzleio/kuzzle/blob/115be4fcfc70156d05af7560c003951de4a00690/lib/api/core/notifier.js#L288): Complete the task associated to this TODO comment. [rule](https://sonarqube.kaliop.net/coding_rules#rule_key=javascript%3AS1135)
1. ![INFO][INFO] [kuzzle.js#L55](https://github.com/kuzzleio/kuzzle/blob/115be4fcfc70156d05af7560c003951de4a00690/lib/api/kuzzle.js#L55): Complete the task associated to this TODO comment. [rule](https://sonarqube.kaliop.net/coding_rules#rule_key=javascript%3AS1135)
1. ![INFO][INFO] [elasticsearch.js#L1562](https://github.com/kuzzleio/kuzzle/blob/115be4fcfc70156d05af7560c003951de4a00690/lib/services/elasticsearch.js#L1562): Complete the task associated to this TODO comment. [rule](https://sonarqube.kaliop.net/coding_rules#rule_key=javascript%3AS1135)
1. ![INFO][INFO] [elasticsearch.js#L1603](https://github.com/kuzzleio/kuzzle/blob/115be4fcfc70156d05af7560c003951de4a00690/lib/services/elasticsearch.js#L1603): Complete the task associated to this TODO comment. [rule](https://sonarqube.kaliop.net/coding_rules#rule_key=javascript%3AS1135)
1. ![INFO][INFO] [elasticsearch.js#L1681](https://github.com/kuzzleio/kuzzle/blob/115be4fcfc70156d05af7560c003951de4a00690/lib/services/elasticsearch.js#L1681): Complete the task associated to this TODO comment. [rule](https://sonarqube.kaliop.net/coding_rules#rule_key=javascript%3AS1135)

[INFO]: https://sonarsource.github.io/sonar-github/severity-info.png 'Severity: INFO'
[MAJOR]: https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR'

**Comment by kuzzle** [comment]
SonarQube analysis reported 9 issues

Note: The following issues were found on lines that were not modified in the pull request. Because these issues can't be reported as line comments, they are summarized here:

1. ![INFO][INFO] [adminController.js#L57](https://github.com/kuzzleio/kuzzle/blob/af6565c40463c77fd6641e9363c3f97c60129116/lib/api/controllers/adminController.js#L57): Complete the task associated to this TODO comment. [rule](https://sonarqube.kaliop.net/coding_rules#rule_key=javascript%3AS1135)
1. ![INFO][INFO] [baseController.js#L648](https://github.com/kuzzleio/kuzzle/blob/af6565c40463c77fd6641e9363c3f97c60129116/lib/api/controllers/baseController.js#L648): Complete the task associated to this TODO comment. [rule](https://sonarqube.kaliop.net/coding_rules#rule_key=javascript%3AS1135)
1. ![INFO][INFO] [documentController.js#L430](https://github.com/kuzzleio/kuzzle/blob/af6565c40463c77fd6641e9363c3f97c60129116/lib/api/controllers/documentController.js#L430): Complete the task associated to this TODO comment. [rule](https://sonarqube.kaliop.net/coding_rules#rule_key=javascript%3AS1135)
1. ![INFO][INFO] [documentController.js#L480](https://github.com/kuzzleio/kuzzle/blob/af6565c40463c77fd6641e9363c3f97c60129116/lib/api/controllers/documentController.js#L480): Complete the task associated to this TODO comment. [rule](https://sonarqube.kaliop.net/coding_rules#rule_key=javascript%3AS1135)
1. ![INFO][INFO] [notifier.js#L288](https://github.com/kuzzleio/kuzzle/blob/af6565c40463c77fd6641e9363c3f97c60129116/lib/api/core/notifier.js#L288): Complete the task associated to this TODO comment. [rule](https://sonarqube.kaliop.net/coding_rules#rule_key=javascript%3AS1135)
1. ![INFO][INFO] [kuzzle.js#L55](https://github.com/kuzzleio/kuzzle/blob/af6565c40463c77fd6641e9363c3f97c60129116/lib/api/kuzzle.js#L55): Complete the task associated to this TODO comment. [rule](https://sonarqube.kaliop.net/coding_rules#rule_key=javascript%3AS1135)
1. ![INFO][INFO] [elasticsearch.js#L1559](https://github.com/kuzzleio/kuzzle/blob/af6565c40463c77fd6641e9363c3f97c60129116/lib/services/elasticsearch.js#L1559): Complete the task associated to this TODO comment. [rule](https://sonarqube.kaliop.net/coding_rules#rule_key=javascript%3AS1135)
1. ![INFO][INFO] [elasticsearch.js#L1601](https://github.com/kuzzleio/kuzzle/blob/af6565c40463c77fd6641e9363c3f97c60129116/lib/services/elasticsearch.js#L1601): Complete the task associated to this TODO comment. [rule](https://sonarqube.kaliop.net/coding_rules#rule_key=javascript%3AS1135)
1. ![INFO][INFO] [elasticsearch.js#L1675](https://github.com/kuzzleio/kuzzle/blob/af6565c40463c77fd6641e9363c3f97c60129116/lib/services/elasticsearch.js#L1675): Complete the task associated to this TODO comment. [rule](https://sonarqube.kaliop.net/coding_rules#rule_key=javascript%3AS1135)

[INFO]: https://sonarsource.github.io/sonar-github/severity-info.png 'Severity: INFO'

**Comment by kuzzle** [comment]
SonarQube analysis reported 9 issues

Note: The following issues were found on lines that were not modified in the pull request. Because these issues can't be reported as line comments, they are summarized here:

1. ![INFO][INFO] [adminController.js#L57](https://github.com/kuzzleio/kuzzle/blob/a7fa0b50c309130a915b78b2016e8440f4026c30/lib/api/controllers/adminController.js#L57): Complete the task associated to this TODO comment. [rule](https://sonarqube.kaliop.net/coding_rules#rule_key=javascript%3AS1135)
1. ![INFO][INFO] [baseController.js#L648](https://github.com/kuzzleio/kuzzle/blob/a7fa0b50c309130a915b78b2016e8440f4026c30/lib/api/controllers/baseController.js#L648): Complete the task associated to this TODO comment. [rule](https://sonarqube.kaliop.net/coding_rules#rule_key=javascript%3AS1135)
1. ![INFO][INFO] [documentController.js#L430](https://github.com/kuzzleio/kuzzle/blob/a7fa0b50c309130a915b78b2016e8440f4026c30/lib/api/controllers/documentController.js#L430): Complete the task associated to this TODO comment. [rule](https://sonarqube.kaliop.net/coding_rules#rule_key=javascript%3AS1135)
1. ![INFO][INFO] [documentController.js#L480](https://github.com/kuzzleio/kuzzle/blob/a7fa0b50c309130a915b78b2016e8440f4026c30/lib/api/controllers/documentController.js#L480): Complete the task associated to this TODO comment. [rule](https://sonarqube.kaliop.net/coding_rules#rule_key=javascript%3AS1135)
1. ![INFO][INFO] [notifier.js#L288](https://github.com/kuzzleio/kuzzle/blob/a7fa0b50c309130a915b78b2016e8440f4026c30/lib/api/core/notifier.js#L288): Complete the task associated to this TODO comment. [rule](https://sonarqube.kaliop.net/coding_rules#rule_key=javascript%3AS1135)
1. ![INFO][INFO] [kuzzle.js#L55](https://github.com/kuzzleio/kuzzle/blob/a7fa0b50c309130a915b78b2016e8440f4026c30/lib/api/kuzzle.js#L55): Complete the task associated to this TODO comment. [rule](https://sonarqube.kaliop.net/coding_rules#rule_key=javascript%3AS1135)
1. ![INFO][INFO] [elasticsearch.js#L1559](https://github.com/kuzzleio/kuzzle/blob/a7fa0b50c309130a915b78b2016e8440f4026c30/lib/services/elasticsearch.js#L1559): Complete the task associated to this TODO comment. [rule](https://sonarqube.kaliop.net/coding_rules#rule_key=javascript%3AS1135)
1. ![INFO][INFO] [elasticsearch.js#L1601](https://github.com/kuzzleio/kuzzle/blob/a7fa0b50c309130a915b78b2016e8440f4026c30/lib/services/elasticsearch.js#L1601): Complete the task associated to this TODO comment. [rule](https://sonarqube.kaliop.net/coding_rules#rule_key=javascript%3AS1135)
1. ![INFO][INFO] [elasticsearch.js#L1675](https://github.com/kuzzleio/kuzzle/blob/a7fa0b50c309130a915b78b2016e8440f4026c30/lib/services/elasticsearch.js#L1675): Complete the task associated to this TODO comment. [rule](https://sonarqube.kaliop.net/coding_rules#rule_key=javascript%3AS1135)

[INFO]: https://sonarsource.github.io/sonar-github/severity-info.png 'Severity: INFO'

**Comment by kuzzle** [comment]
SonarQube analysis reported 9 issues

Note: The following issues were found on lines that were not modified in the pull request. Because these issues can't be reported as line comments, they are summarized here:

1. ![INFO][INFO] [adminController.js#L57](https://github.com/kuzzleio/kuzzle/blob/48ddbeb3719a79114fe412a9ec7b7687b38f1f24/lib/api/controllers/adminController.js#L57): Complete the task associated to this TODO comment. [rule](https://sonarqube.kaliop.net/coding_rules#rule_key=javascript%3AS1135)
1. ![INFO][INFO] [baseController.js#L648](https://github.com/kuzzleio/kuzzle/blob/48ddbeb3719a79114fe412a9ec7b7687b38f1f24/lib/api/controllers/baseController.js#L648): Complete the task associated to this TODO comment. [rule](https://sonarqube.kaliop.net/coding_rules#rule_key=javascript%3AS1135)
1. ![INFO][INFO] [documentController.js#L430](https://github.com/kuzzleio/kuzzle/blob/48ddbeb3719a79114fe412a9ec7b7687b38f1f24/lib/api/controllers/documentController.js#L430): Complete the task associated to this TODO comment. [rule](https://sonarqube.kaliop.net/coding_rules#rule_key=javascript%3AS1135)
1. ![INFO][INFO] [documentController.js#L480](https://github.com/kuzzleio/kuzzle/blob/48ddbeb3719a79114fe412a9ec7b7687b38f1f24/lib/api/controllers/documentController.js#L480): Complete the task associated to this TODO comment. [rule](https://sonarqube.kaliop.net/coding_rules#rule_key=javascript%3AS1135)
1. ![INFO][INFO] [notifier.js#L288](https://github.com/kuzzleio/kuzzle/blob/48ddbeb3719a79114fe412a9ec7b7687b38f1f24/lib/api/core/notifier.js#L288): Complete the task associated to this TODO comment. [rule](https://sonarqube.kaliop.net/coding_rules#rule_key=javascript%3AS1135)
1. ![INFO][INFO] [kuzzle.js#L55](https://github.com/kuzzleio/kuzzle/blob/48ddbeb3719a79114fe412a9ec7b7687b38f1f24/lib/api/kuzzle.js#L55): Complete the task associated to this TODO comment. [rule](https://sonarqube.kaliop.net/coding_rules#rule_key=javascript%3AS1135)
1. ![INFO][INFO] [elasticsearch.js#L1559](https://github.com/kuzzleio/kuzzle/blob/48ddbeb3719a79114fe412a9ec7b7687b38f1f24/lib/services/elasticsearch.js#L1559): Complete the task associated to this TODO comment. [rule](https://sonarqube.kaliop.net/coding_rules#rule_key=javascript%3AS1135)
1. ![INFO][INFO] [elasticsearch.js#L1601](https://github.com/kuzzleio/kuzzle/blob/48ddbeb3719a79114fe412a9ec7b7687b38f1f24/lib/services/elasticsearch.js#L1601): Complete the task associated to this TODO comment. [rule](https://sonarqube.kaliop.net/coding_rules#rule_key=javascript%3AS1135)
1. ![INFO][INFO] [elasticsearch.js#L1675](https://github.com/kuzzleio/kuzzle/blob/48ddbeb3719a79114fe412a9ec7b7687b38f1f24/lib/services/elasticsearch.js#L1675): Complete the task associated to this TODO comment. [rule](https://sonarqube.kaliop.net/coding_rules#rule_key=javascript%3AS1135)

[INFO]: https://sonarsource.github.io/sonar-github/severity-info.png 'Severity: INFO'

**Pull request merged**